src/containers/home/components/use-fetch-version-profiles.tsx

```diff
@@ -0,0 +1,59 @@
+import Axios from 'axios';
+import { useEffect, useState } from 'react';
+import { TitlesEntity } from '../../../types/overview';
+import { getTitle } from '../utils/title';
+
+type Value = {
+  titles: string;
+};
+
+export type Model = {
+  search: string;
+  title: string;
+  value: Value[];
+};
+
+function UseFetchProfiles(version: string, path: string) {
+  const [modelSearchData, setModelSearchData] = useState<Array<any>>([]);
+  const [error, setError] = useState(null);
```

Comment:
the type of this error will be `AxiosError`, do you want to return that from this hook? 
Also I noticed this error isn't used anywhere.

Maybe we can show a snackbar (using [notistack](https://material-ui.com/components/snackbars/#notistack)) which gives more context to the user

Reply:
same goes for other fetch hooks

Reply:
I thought there would be no harm in returning any exception out of 2xx in case it's necessary to catch them in the future. Any thoughts on why we should refrain from doing so?